### PR TITLE
ONYX-11826 Reorganise configuration 

### PR DESCRIPTION
**Pull request by doodlesbykumbi**
### What does this PR do?
- _What's changed? Why were these changes made?_
- _How should the reviewer approach this PR, especially if manual tests are required?_
- _Are there relevant screenshots you can add to the PR description?_

### What ticket does this PR close?
Resolves #[relevant GitHub issues, eg 76]

#### Change log
- [ ] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [ ] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [ ] This PR does not require updating any documentation

### CODE REVIEW

File: pkg/secrets/config/config_test.go [hunk 1]
```diff
@@ -35,11 +35,15 @@ func TestConfig(t *testing.T) {
...
-						RequiredK8sSecrets: testK8sSecretsList,
-						RetryIntervalSec:   testRetryIntervalSec,
-						StoreType:          storeType,
+						ContainerConfig: ContainerConfig{
+							PodNamespace:       testNamespace,
+							RetryCountLimit:    testRetryCountLimit,
+							RequiredK8sSecrets: testK8sSecretsList,
+							RetryIntervalSec:   testRetryIntervalSec,
+						},
+						SecretsConfig: SecretsConfig{
```
Comment (sigalsax) [inline pkg/secrets/config/config_test.go:44]: do we need to add or change tests ?

Comment (doodlesbykumbi) [reply pkg/secrets/config/config_test.go:44]: I don't believe so. The tests that exist seem to be covering the config well

**Review by diverdane — APPROVED** [review]
LGTM!
I'm approving, but I don't know if we still need this change?

**Comment by jtuttle** [comment]
We closed the clone of this issue at standup today and decided not to do it yet.

**Pull request closed without merging**